apps/charterafrica/src/payload/blocks/MediaBlock.js

```diff
@@ -0,0 +1,20 @@
+import blockFields from "../fields/blockFields";
+
+const MediaBlock = {
+  slug: "mediaBlock",
+  fields: [
+    blockFields({
+      name: "mediaBlockFields",
+      fields: [
+        {
+          name: "media",
+          type: "upload",
+          relationTo: "media",
+          required: true,
```

Comment:
Do we allow any other media than image? Can front-end handle anything other than image?

Reply:
we can add a filter

Reply:
Cool... lets also rename it to `image` if we're only supporting images.